library/Icingadb/Widget/Detail/CheckStatistics.php

```diff
@@ -141,6 +142,7 @@ protected function assembleBody(BaseHtmlElement $body)
 
     protected function assembleFooter(BaseHtmlElement $footer)
     {
+        $footer->add(new HorizontalKeyValue(t('Scheduling source'), $this->object->state->scheduling_source));
```

Comment:
* Capitalize `source`. You did that in the event detail as well. Otherwise we'll get two distinct messages in the translation catalog
  * While you're at it, do the same for the `Check source` label. It has the same problem already
* Show a colon after the label. That makes it obvious that the label identifies something and that something isn't part of the label
  * Add the colon not to the localized string directly, but concatenate it with the result. It's more of a structural element than a part of the label